src/audio/mixin_mixout.c

```diff
@@ -886,11 +886,9 @@ static int mixout_copy(struct comp_dev *dev)
 		mixin = buffer_get_comp(unused_in_between_buf, PPL_DIR_UPSTREAM);
 
 		src_info = find_mixout_source_info(mixed_data_info, mixin);
-		if (!src_info) {
-			comp_err(dev, "No source info");
-			mixed_data_info_release(mixed_data_info);
-			return -EINVAL;
-		}
+		/* this shouldn't happen but skip even if it does and move to the next source */
+		if (!src_info)
```

Comment:
should we keep an error message if this should never happen?

Reply:
i don't know. maybe a warning but that will only pollute the logs I think.